.golangci.yml

```diff
@@ -0,0 +1,131 @@
+---
+run:
+  build-tags:
+    - netgo
+  concurrency: 6
+  deadline: 5m
+linters:
+  disable-all: true
+  enable:
+    - asciicheck
+    - bodyclose
+    - deadcode
+    - depguard
+    - dogsled
+    - dupl
+    - errcheck
+    - gochecknoinits
+    - goconst
+    - gocritic
+    - gocyclo
+    - gofmt
+    - goimports
+    - golint
+    - goprintffuncname
+    - gosimple
+    - govet
+    - ineffassign
+    - interfacer
+    - maligned
+    - misspell
+    - nakedret
+    - nolintlint
+    - prealloc
+    - rowserrcheck
+    - staticcheck
+    - structcheck
+    - stylecheck
+    - typecheck
+    - unconvert
+    - unparam
+    - unused
+    - varcheck
+    - whitespace
+    - gocognit
+    - godox
+    - gomnd
+    - gomodguard
+    - gosec
+    - lll
+    - nestif
+    # - funlen
```

Comment:
Do we think we will want to enable these commented out rules in the future?

Reply:
Yes, I wanted to follow up to fix some of those as well. :)